tests/test_safety.py

```diff
@@ -220,6 +221,50 @@ def test_multiple_versions(self):
         )
         self.assertEqual(len(vulns), 4)
 
+    @patch.object(requests, 'get', side_effect=requests.exceptions.ConnectionError)
+    def test_check_fetch_database_url_connection_error(self, requests_mock):
+        from safety.errors import NetworkConnectionError
+
+        db_name = "insecure.json"
+        mirror = 'https://safety.test'
+
+        with self.assertRaises(NetworkConnectionError):
+            safety.fetch_database_url(mirror, db_name=db_name, key="INVALID", cached=False, proxy={})
+
```

Comment:
Thanks for the tests. :clap: 

Codecov provides some line-by-line comments as part of our PRs about test coverage. Please make sure we are watching them and respecting as much as possible.